.github/workflows/python.yml

```diff
@@ -7,7 +7,8 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    # `ubuntu-22.04` doesn't support Python 3.7 anymore
```

Comment:
The commit message and this comment seem to contradict each other ...

Reply:
Fixed in commit 6bcfc1cce4e4bb03e4e48ea818f357b3b36d9593.